src/main/java/com/example/bookshop/controller/BookController.java

```diff
@@ -29,25 +30,29 @@
 public class BookController {
     private final BookService bookService;
 
+    @PreAuthorize("hasRole('ROLE_USER')")
```

Comment:
Request to this endpoint can be sent not only from user with USER role. But to requirements LGTM